zipkin-example/src/main/java/com/example/helloworld/HelloWorldApplication.java

```diff
@@ -43,6 +45,9 @@ public void run(HelloWorldConfiguration configuration,
         final Client client = new ZipkinClientBuilder(environment, brave)
                 .build(configuration.getZipkinClient());
 
```

Comment:
thanks for the effort.. I think you should carry-on as adding the first
tests are always hardest, but benefit all future contributors (as all they
need to do is add another test vs figure out how to test).

Of course, this effort could go on as a separate issue, but definitely if
you are on a roll, keep rolling!